recommendation/system.web.json

```diff
@@ -116,21 +129,321 @@
+            {
+              "Name": "netcoreapp3.1",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            },
+            {
+              "Name": "net5.0",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            }
+          ],
+          "Description": "Remove IHttpHandler in namespace System.Web and replace HttpContext in namespace Microsoft.AspNetCore.Http.",
+          "Actions": [
+            {
+              "Name": "RemoveDirective",
+              "Type": "Using",
+              "Value": "System.Web",
+              "Description": "Remove System.Web namespace.",
+              "ActionValidation": {
+                "Contains": "",
+                "NotContains": "using System.Web;"
+              }
+            },
+            {
+              "Name": "AddDirective",
+              "Type": "Using",
+              "Value": "Microsoft.AspNetCore.Http",
+              "Description": "Add Microsoft.AspNetCore.Http namespace.",
+              "ActionValidation": {
+                "Contains": "using Microsoft.AspNetCore.Http;",
+                "NotContains": ""
+              }
+            },
+            {
+              "Name": "AddDirective",
+              "Type": "Using",
+              "Value": "System.Threading.Tasks",
+              "Description": "Add System.Threading.Tasks namespace",
+              "ActionValidation": {
+                "Contains": "using System.Threading.Tasks;",
+                "NotContains": ""
+              }
+            },
+            {
+              "Name": "RemoveBaseClass",
+              "Type": "Class",
+              "Value": "IHttpModule",
+              "Description": "Remove the base class IHttpModule",
+              "ActionValidation": {
+                "Contains": "",
+                "NotContains": ":IHttpModule"
+              }
+            },
+            {
+              "Name": "CreateConstructor",
+              "Type": "Class",
+              "Value": {
+                "types": "RequestDelegate",
+                "identifiers": "next"
+              },
+              "Description": "Create a new constructor"
+            },
+            {
+              "Name": "AppendConstructorExpression",
+              "Type": "Class",
+              "Value": "_next = next;",
+              "Description": "Add a new expression in the constructor to initialize the new variable of _next to the next element in the pipeline.",
+              "ActionValidation": {
+                "Contains": "_next = next;",
+                "NotContains": ""
+              }
+            },
+            {
+              "Name": "AddExpression",
+              "Type": "Class",
+              "Value": "RequestDelegate _next = null;",
+              "Description": "Add a new global variable to replace the OwinMiddleware global abstract variable of Next.",
+              "ActionValidation": {
+                "Contains": "RequestDelegate_next=null;",
```

Comment:
spaces here again

Reply:
Although we do not need spaces, let's be consistent and add them anyway for readability and consistency if that's ok.